test/e2e/deferred/restart.go

```diff
@@ -20,6 +22,23 @@ import (
 	"github.com/openshift/cluster-node-tuning-operator/test/e2e/util/wait"
 )
 
+const (
+	enableRestartTestsEnvVar = "DEFERRED_RESTART_TESTS"
+)
+
+func areRestartTestsEnabled() bool {
+	envVal, ok := os.LookupEnv(enableRestartTestsEnvVar)
+	if !ok {
+		return false
+	}
+	val, err := strconv.ParseBool(envVal)
+	if err != nil {
+		util.Logf("wrong value for %q: %q (err=%v)", enableRestartTestsEnvVar, envVal, err)
+		return false
+	}
+	return val
+}
+
 var _ = ginkgo.Describe("[deferred][restart] Profile deferred", func() {
```

Comment:
Maybe we should also add `[disruptive]` ? 
https://github.com/kubernetes/community/blob/0566459c2bdfec36670861be20be26d33f31467e/contributors/devel/sig-testing/e2e-tests.md#kinds-of-tests

Reply:
indeed, done